spec/classes/mysql_server_backup_spec.rb

```diff
@@ -109,7 +122,7 @@
 
           it 'should be able to backup events table' do
             is_expected.to contain_file('mysqlbackup.sh').with_content(
-              /EVENTS="--events"/
+              /ADDITIONAL_OPTIONS="--events"/
```

Comment:
This is part of my change.

In the mysqlbackup.sh.erb file, there used to be a variable named ```EVENTS``` that just stored the mysqldump options for events.  I've changed this variable to ```ADDITIONAL_OPTIONS```, and am using it to store the events options, as well as the trigger and routine options.